src/python/pants/task/task.py

```diff
@@ -121,12 +121,12 @@ def supports_passthru_args(cls):
   def register_options(cls, register):
     super().register_options(register)
     if cls.supports_passthru_args():
-      register('--passthrough-args',
+      register('--passthru-args',
```

Comment:
I'm going to drop this.

Your codebase uses both spellings, the latter spelling is dominant, but this is effectively an api.